src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputUnderscoreUsedInNames.java

```diff
@@ -22,95 +22,102 @@ public class InputUnderscoreUsedInNames {
   private final String gradle851 = "";
 
   class InnerBad {
-    String guava_33_4_7 = ""; // violation due to '_' between digit and letter
+    String guava_33_4_7 = ""; // violation 'Member name 'guava_33_4_7' must match pattern'
```

Comment:
we already discussed this few times in this PR.
there is not much value in violation message in such complicated style violation. messages is same.
It is better to keep reason on why there is violation.

you can keep both 
https://github.com/checkstyle/checkstyle/blob/f486dcacb9b6096bd28ce9b6b3d215f922f315e7/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L82

`String guava_33_4_7; // violation, due to '_' between digit and letter 'must match pattern'`

Please apply to all.